docker/packager/binary/Dockerfile

```diff
@@ -24,30 +24,25 @@ RUN apt-get update \
     && apt-key add /tmp/llvm-snapshot.gpg.key \
     && export CODENAME="$(lsb_release --codename --short | tr 'A-Z' 'a-z')" \
     && echo "deb [trusted=yes] https://apt.llvm.org/${CODENAME}/ llvm-toolchain-${CODENAME}-${LLVM_VERSION} main" >> \
-        /etc/apt/sources.list
+        /etc/apt/sources.list \
+    && apt-get clean
 
 # initial packages
-RUN apt-get update \
-    && apt-get install \
-        bash \
-        fakeroot \
-        ccache \
-        curl \
-        software-properties-common \
-        --yes --no-install-recommends
-
 RUN apt-get update \
     && apt-get install \
         bash \
         build-essential \
         ccache \
+        clang-${LLVM_VERSION} \
+        clang-tidy-${LLVM_VERSION} \
         cmake \
         curl \
+        fakeroot \
         gdb \
         git \
         gperf \
-        clang-${LLVM_VERSION} \
-        clang-tidy-${LLVM_VERSION} \
+        libicu-dev \
```

Comment:
Why we need them? We have bundled versions in contrib.

Reply:
It looks like some strange merge back and forth issues to me. Should I just remove both readline and libicu?

Reply:
Actually, just have it done. Let's see how it goes

Reply:
https://s3.amazonaws.com/clickhouse-test-reports/32415/9a6e8dc61505b6bf9b27c61fc56e4fa257653aa6/fast_test__actions_.html

Fast tests passed, so maybe you are right